ucsb-cs156-s24/team03-s24-4pm-8
# Rk indexpage and editpage

https://ucsb-cs156-s24.github.io/team03-s24-4pm-8/prs/98/storybook
Implemented both Index and Edit page in this pr.

to test:

edit functionality:
<img width="1396" alt="Screenshot 2024-05-13 at 4 14 26 PM" src="https://github.com/ucsb-cs156-s24/team03-s24-4pm-8/assets/92010151/089ca89a-339b-4f43-868e-e0dfb2adeaeb">

click on the edit button and change the content as you please. hit submit and your changes should show up on the page:
<img width="1392" alt="Screenshot 2024-05-13 at 4 14 56 PM" src="https://github.com/ucsb-cs156-s24/team03-s24-4pm-8/assets/92010151/f978848a-d635-4ed3-8066-0b2addc9e9e7">

<img width="1362" alt="Screenshot 2024-05-13 at 4 15 00 PM" src="https://github.com/ucsb-cs156-s24/team03-s24-4pm-8/assets/92010151/68662156-1e42-48a3-b9f9-5b04e89dadb1">

delete functionality:
select which item you would like to delete and hit delete. the item should not show up on the refreshed page
<img width="1420" alt="Screenshot 2024-05-13 at 4 21 46 PM" src="https://github.com/ucsb-cs156-s24/team03-s24-4pm-8/assets/92010151/844fc79e-a05f-4eeb-9cf0-af2bdbb9034a">

<img width="1361" alt="Screenshot 2024-05-13 at 4 21 51 PM" src="https://github.com/ucsb-cs156-s24/team03-s24-4pm-8/assets/92010151/be660cad-44ff-4903-98c4-f8b0bacbb048">



closes #15 
closes #14
